source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -140,6 +140,28 @@ export class Carousel extends Container {
         }
     }
 
+    static readonly initialPageProperty = new NumProperty(Versions.v1_6, "initialPage", 0);
+    @property(Carousel.initialPageProperty)
+    get initialPage(): number {
+        let initialPageIndex = this.getValue(Carousel.initialPageProperty); 
+
+        if (initialPageIndex < 0) { 
```

Comment:
Should we also check the upper bound here?

Reply:
I was delegating the task to swiper module to handle since it does handle the way we want, but your point is good. I updated the code to validate these, such that we can provide good info to users when things go wrong.